### PR TITLE
I create the portfolio and about me sections for the Skeleton Project

### PR DESCRIPTION
**Pull request by nmeziany**
In this project I create:
- Portfolio section to present the projects created by the owner of the portfolio.
- About me section to allow the contact with the portfolio owner via several social networks accounts.

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -0,0 +1,265 @@
...
+  margin-top: 0;
+}
+
+li {
+  display: flex;
+  padding-right: 20px;
+}
+
+.boxworks {
+  display: flex;
```
Comment (Lordkaito) [inline style.css]: - [x] Please make sure to use `display: grid` for this section, as this is one of the requirements of this project. Your result it's good, but as I mentioned, you have to follow the requirements given for this project, and `grid` it's one of them, so please apply the necessary changes.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,205 @@
...
+            <li class="lang">css</li>
+            <li class="lang">javaScript</li>
+          </ul>
+          <a class="moreabout" href="#">See Project</a>
+        </div>
+      </section>
+
+      <!-- About me section! -->
+      <section class="aboutme">
+        <div class="title">About Myself</div>
```
Comment (Lordkaito) [inline index.html]: - [x] Kindly change this `div` into an `h1` tag, as this text it's actually a title, let's try to keep it as semantic as possible!

File: index.html [hunk 3]
```diff
@@ -0,0 +1,205 @@
...
+        <div class="txt">
+          Hello I’m a software developer! I can help you build a product ,
+          feature or website Look through some of my work and experience! If you
+          like what you see and have a project you need coded, don’t hestiate to
+          contact me.
```
Comment (Lordkaito) [inline index.html:155]: - [x] Please make sure to wrap this text into a `p` tag, as this is good for semantic purposes as well, so go ahead with those changes!

File: index.html [hunk 4]
```diff
@@ -0,0 +1,205 @@
...
+              <a href="#"><img src="images/Y.svg" alt="" /></a>
+            </li>
+          </ul>
+          <div class="moreabout"><a href="#">Get my resume</a></div>
+        </div>
+        <div class="resume">
+          <div>
+            <h2 class="subt">Languages</h2>
+            <img src="images/works/Uniona.png" alt="" />
+          </div>
```
Comment (Lordkaito) [inline index.html:182]: - [x] Please make sure to wrap all "Languages", "Frameworks" and "Skills" into a `ul` tag, with the correspondent `li` tag each. As a result of this change, you will have 2 nested `ul`'s, one wrapping "Frameworks, Languages and Skills" and the other one wrapping "HTML", "CSS" and "JavaScript".

**Review by Lordkaito — CHANGES_REQUESTED** [review]
Hi @nmeziany, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: style.css (hunk 1 above)
Comment (nmeziany) [reply style.css]: Done!

File: index.html (hunk 2 above)
Comment (nmeziany) [reply index.html]: Changes done!

File: index.html (hunk 3 above)
Comment (nmeziany) [reply index.html:155]: p tag added

File: index.html (hunk 4 above)
Comment (nmeziany) [reply index.html:182]: done!

**Review by Lordkaito — APPROVED** [review]
Hi @nmeziany, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**